client/llb/state.go

```diff
@@ -320,6 +320,14 @@ func (s State) User(v string) State {
 	return User(v)(s)
 }
 
+func (s State) Hostname(v string) State {
```

Comment:
Missing `func Hostname(str string) StateOption` variant that can be passed to `Run`

Reply:
Hi @tonistiigi 

This func is put in client/llb/meta.go L147, is it the right place?

Reply:
looks good